recipes/hyperts/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "HyperTS" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/DataCanvasIO/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: e5274e2398412220b532be8e5f9bd3baad51b8538e536ba3b7494aabc03bd042
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6, <3.10
+    - pip
+  run:
```

Comment:
Upstream lists Tensorflow, is that not required?

https://github.com/DataCanvasIO/HyperTS/blob/90334143d6fbe966a35be57941c3e8ee55e7755f/setup.py#L53

Reply:
It needs `tensorflow>=2.0`, but the version is not released on windows, however we hope to use it on windows.
So we changed tensorflow as a optional item , in this way, it can run without installing tensorflow, at the cost that the relevant features cannot be used.
Users can manually install (use pip or conda if possible ) tensorflow to use that features.